### ISSUE TITLE
investigating creating minimal release version

### ISSUE THREAD

**Issue by davidwallis3101** [Bounz/HomeGenie-BE#23]
no scheduled tasks, no example programs, no pre-installed interfaces etc.

**Comment by tuicemen** [comment]
Stripping everything out creates a stupid server.
This is fine for experianced user but i feel it would reduce the number of new users. Perhaps  if this is done a installer could be created that asked users what they wished to initialy install.

**Comment by Bounz** [comment]
Yes, of course. Demo data will remain in some form. I think there will be a promt after a fresh install with options to restore backup or create demo data.
